streams/examples/main/java/org/apache/kafka/streams/examples/wordcount/WordCountJob.java

```diff
@@ -63,11 +64,10 @@ public static void main(String[] args) throws Exception {
         final Deserializer<String> stringDeserializer = new StringDeserializer();
         final Serializer<Long> longSerializer = new LongSerializer();
         final Deserializer<Long> longDeserializer = new LongDeserializer();
-        final Serializer<JsonNode> JsonSerializer = new JsonSerializer();
 
         KStream<String, String> source = builder.stream("streams-file-input");
 
-        KStream<String, JsonNode> counts = source
+        KTable<String, Long> counts = source
```

Comment:
Oh, my bad. I should not use any lambdas actually since by default it is compiled in Java 7 not Java 8. Will remove all lambdas expression.